app/src/main/java/com/babylon/wallet/android/utils/AppEventBusImpl.kt

```diff
@@ -7,15 +7,13 @@ import kotlinx.coroutines.flow.Flow
 import kotlinx.coroutines.flow.MutableSharedFlow
 import kotlinx.coroutines.flow.asSharedFlow
 import javax.inject.Inject
-import javax.inject.Singleton
 
 interface AppEventBus {
 
     val events: Flow<AppEvent>
     suspend fun sendEvent(event: AppEvent, delayMs: Long = 0L)
 }
 
-@Singleton
 class AppEventBusImpl @Inject constructor() : AppEventBus {
```

Comment:
also the  `@Inject constructor()`

Reply:
you have already a hilt binding

Reply:
anyway it is not necessary , all good 🆗

Reply:
didn't know that @Inject constructor is not needed here

Reply:
it's minor, it doesnt affect anything in hilt.

Reply:
my bad. forget this comment 😓